### PR TITLE
interfaces/network: allow using netcat as client

### PR DESCRIPTION
**Pull request by alfonsosanchezbeato**
Add permissions to run netcat to network interface

### CODE REVIEW

**Comment by alfonsosanchezbeato** [comment]
@jdstrand alright, makes sense. Thanks!

**Comment by alfonsosanchezbeato** [comment]
@mvo5 @jdstrand @chipaca it turns out that by using network+network-observe+hardware-observe, the only additional change that is needed is adding permissions to run netcat in the network interface. Branch re-pushed with that change, and ready for review again.

**Comment by niemeyer** [comment]
@jdstrand This is ready for re-review.

File: interfaces/builtin/network.go [hunk 1]
```diff
@@ -56,6 +56,9 @@ dbus send
 
 @{PROC}/sys/net/core/somaxconn r,
 @{PROC}/sys/net/ipv4/tcp_fastopen r,
+
+# Allow using netcat as client
+/bin/nc.openbsd ixr,
```
Comment (jdstrand) [inline interfaces/builtin/network.go]: The rule as it stands is fine, however we should try to be cross-distro friendly. Please use this instead:
```
/{,usr/}bin/nc{,.openbsd} ixr,
```

**Comment by mvo5** [comment]
I did the (trivial) update that Jamie asked for, let see if tests are happy today.

**Comment by alfonsosanchezbeato** [comment]
@mvo5 thanks for pushing the change, it looks also that all checks have passed.

**Pull request merged**